packages/mui-system/package.json

```diff
@@ -41,20 +41,20 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.22.15",
-    "@mui/private-theming": "^5.14.8",
-    "@mui/styled-engine": "^5.14.8",
+    "@mui/private-theming": "^5.14.9",
+    "@mui/styled-engine": "^5.14.9",
     "@mui/types": "^7.2.4",
-    "@mui/utils": "^5.14.8",
+    "@mui/utils": "^5.14.9",
     "clsx": "^2.0.0",
     "csstype": "^3.1.2",
     "prop-types": "^15.8.1"
   },
   "devDependencies": {
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
-    "@mui/joy": "5.0.0-beta.5",
-    "@mui/material": "^5.14.8",
-    "@mui/material-next": "6.0.0-alpha.100",
+    "@mui/joy": "5.0.0-beta.6",
+    "@mui/material": "^5.14.9",
+    "@mui/material-next": "6.0.0-alpha.101",
```

Comment:
Off-topic: not sure why the system has them as dev dependencies 🤔

Reply:
The dependencies are now explicit, we use them in the tests.

Reply:
I would challenge that having this here is a net-positive: https://github.com/mui/material-ui/pull/38859#discussion_r1324605245.